### PR TITLE
FT-139: workflowTemplate to upgrade a runtime

### PR DESCRIPTION
**Pull request by lrochette**

### CODE REVIEW

File: examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        - name: CF_HOST_URL
+          default: https://g.codefresh.io
+        - name: GIT_SECRET
+          default: 'github-token'
+        - name: GIT_TOKEN_SECRET_KEY
+          default: 'token'
+      container:
+        # name: main
+        image: quay.io/codefresh/cli-v2
+        # imagePullPolicy: Always
```
Comment (eti-codefresh) [inline examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml:44]: it this suppose to stay like this?

File: examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        - name: CF_TOKEN_SECRET_KEY
+          default: token
+        - name: CF_HOST_URL
+          default: https://g.codefresh.io
+        - name: GIT_SECRET
+          default: 'github-token'
+        - name: GIT_TOKEN_SECRET_KEY
+          default: 'token'
+      container:
+        # name: main
```
Comment (eti-codefresh) [inline examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml:42]: it this suppose to stay like this?

File: examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    argo-hub/version: '0.0.1'
+    argo-hub/description: 'This Workflow Template is used to upgrade a GitOps Runtime'
+    argo-hub/categories: 'Administration'
+    argo-hub/license: 'MIT'
+    argo-hub/owner_name: 'Laurent Rochette'
+    argo-hub/owner_email: 'laurent.rochette@codefresh.io'
+    argo-hub/owner_avatar: 'https://avatars.githubusercontent.com/lrochette'
+    argo-hub/owner_url: 'https://github.com/lrochette'
+    argo-hub/icon_url: "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/upgrade-cluster/assets/icon.svg"
+    argo-hub/icon-background: "#004C76"
```
Comment (eti-codefresh) [inline examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml:15]: argo-hub/icon_background

**Review by eti-codefresh — APPROVED** [review]

File: examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml (hunk 1 above)
Comment (lrochette) [reply examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml:44]: No. I was doing some debugging as I thought my code was incorrect so I commented a bunch of stuff and then I realized later that the container name was not required

File: examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml (hunk 3 above)
Comment (lrochette) [reply examples/upgrade-cluster/versions/0.0.1/workflowTemplate.yaml:15]: Fixed

**Pull request merged**